### ISSUE TITLE
SolutionParser is crashed and previewer is not loaded correctly.

### ISSUE THREAD

**Issue by SignalMQ** [AvaloniaUI/AvaloniaVSCode#86]
### Describe the bug

**Hi everyone**
After upgrading this extension version to `0.0.26`, I have installed `.NET 8.0` manually by standalone installer in my system, but this extension is requested to install 8.0 anyway. I fixed this by adding some parameter for `.vscode/settings.json` file:
```json
{
    "dotnetAcquisitionExtension.existingDotnetPath": [
        {
            "extensionId": "msazurermtools.azurerm-vscode-tools", 
            "path": "C:\\Program Files\\dotnet\\dotnet.exe"
        }
    ]
}
```
Can this be automated by an extension?

**My second issue:**
After I clicked to preview button, extension give me message to build project for starting previewer but stuck on starting `SolutionParser.dll` inside `vscode-avalonia` extension, because is this run by 7.0.

I found this in https://github.com/AvaloniaUI/AvaloniaVSCode/blob/70aa1c9e899a9bf37ff8bede1ba930531ea42288/src/vscode-avalonia/src/services/solutionParser.ts#L102C96-L102C107

### To Reproduce

1) Make `Avalonia App` project from **Create Avalonia Project** button
2) Open output logs (my case: it's open by sliding blue line to top at bottom bar)
3) Open `MainWindow.axaml`
4) Press preview button
5) Apply build suggestion

### Avalonia for VS Code

0.0.26

### Avalonia version

11.0.6

### VS Code version

1.85.1

### Relevant log output

```shell
2024-01-08 02:15:41.224 [info] [Info  - 2:15:41 AM] AvaloniaLanguageServer.Handlers.TextDocumentSyncHandler: ** DidOpenText: file:///c:/Users/Sobirjon/Desktop/Projects/TodoApp/MainWindow.axaml | Uri='file:///c:/Users/Sobirjon/Desktop/Projects/TodoApp/MainWindow.axaml'
2024-01-08 02:15:49.210 [info] Command avalonia.previewProcess, file:///c%3A/Users/Sobirjon/Desktop/Projects/TodoApp/MainWindow.axaml
2024-01-08 02:15:55.503 [info] [EXT - INFO] Solution data path path: C:\Users\Sobirjon\AppData\Local\Temp\TodoApp.json
2024-01-08 02:15:55.561 [info] parser process args: C:\Windows\system32\cmd.exe,/d,/s,/c,"dotnet "c:\Users\Sobirjon\.vscode\extensions\avaloniateam.vscode-avalonia-0.0.26\solutionParserTool\SolutionParser.dll" "c:\Users\Sobirjon\Desktop\Projects\TodoApp" --sdk 7.0"
2024-01-08 02:15:59.055 [info] parser process exited 4294967295
```

### Additional context

```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <Nullable>enable</Nullable>
    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
    <ApplicationManifest>app.manifest</ApplicationManifest>
    <AvaloniaUseCompiledBindingsByDefault>true</AvaloniaUseCompiledBindingsByDefault>
  </PropertyGroup>


  <ItemGroup>
    <PackageReference Include="Avalonia" Version="11.0.6" />
    <PackageReference Include="Avalonia.Desktop" Version="11.0.6" />
    <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.6" />
    <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.6" />
    <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.-->
    <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.6" />
  </ItemGroup>
</Project>
```

**Comment by prashantvc** [comment]
Fixed in the v0.0.27 please reopen if you continue to have an issue

**Comment by ezexe** [comment]
im having this issue on macos
